### PR TITLE
feat(file): 2536 show archive sub id & revision

### PR DESCRIPTION
**Pull request by cy**
Closes #2536

### CODE REVIEW

File: app/scripts/files/templates/file.html [hunk 1]
```diff
@@ -61,15 +61,17 @@ <h3 class="panel-title pull-left" data-translate>File Properties</h3>
...
-                  <span data-ng-if="fc.file.archive">
-                    (<a data-ui-sref="search.files({ 'filters': {{
+                <td data-ng-if="fc.file.archive">
+                  {{ ::fc.file.archive.submitter_id || '--' }}
+                  - rev {{ ::fc.file.archive.revision || '--'}}
+                  (<a data-ui-sref="search.files({ 'filters': {{
                       [{ field: 'files.archive.archive_id', value: fc.file.archive.archive_id }] | makeFilter }}
                       })">
                     {{ fc.archiveCount }} files
                   </a>)
```
Comment (shanewilson) [inline app/scripts/files/templates/file.html]: don't link if there are 0 files

Comment (cy) [inline app/scripts/files/templates/file.html]: moved the ng-if to the td level if there are 0 files it'll not show this and show https://github.com/NCI-GDC/portal-ui/pull/1014/files#diff-3acfaf0591334ca8b64227bfc57a39d5R73

**Comment by shanewilson** [comment]
👍

**Comment by ithinkicancode** [comment]
👍

**Pull request merged**